api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/TensorFlowInferenceModel.kt

```diff
@@ -28,6 +31,9 @@ import java.util.*
  * Provides functionality to make predictions and model loading.
  */
 public open class TensorFlowInferenceModel : InferenceModel() {
+    /** The layers to describe the model design. Main part of the internal state of the model. */
+    public lateinit var layers: List<Layer>
```

Comment:
don't like this change, to get access to refactored data in `isOptimizerNameAndRelatedToFrozenLayer` and `getVariablesAndTensors` layers required, will think about it